### PR TITLE
Follow-up regarding GH Actions updates

### PR DESCRIPTION
**Pull request by afalhambra**
Follows up https://github.com/quarkus-qe/beefy-scenarios/pull/307

Updating below github actions:

From `DeLaGuardo/setup-graalvm@master` to `graalvm/setup-graalvm@v1`
From `jitterbit/get-changed-files@v1` to `tj-actions/changed-files@v34.4.3`

### CODE REVIEW

**Comment by afalhambra** [comment]
Blocks https://github.com/quarkus-qe/beefy-scenarios/pull/308

File: .github/workflows/daily.yaml [hunk 1]
```diff
@@ -199,13 +199,14 @@ jobs:
         shell: bash
         run: tar -xzf maven-repo.tgz -C ~
       - name: Install cl.exe
-        uses: ilammy/msvc-dev-cmd@v1
```
Comment (michalvavrik) [inline .github/workflows/daily.yaml:202]: Please see https://github.com/ilammy/msvc-dev-cmd/tags, both ilammy/msvc-dev-cmd@v1 and ilammy/msvc-dev-cmd@v1.12.0 are tags for commit [7315a94](https://github.com/ilammy/msvc-dev-cmd/commit/7315a94840631165970262a99c72cfb48a65d25d). Why is this change needed?

File: .github/workflows/daily.yaml [hunk 2]
```diff
@@ -199,13 +199,14 @@ jobs:
...
-      - uses: microsoft/setup-msbuild@v1
+        uses: ilammy/msvc-dev-cmd@v1.12.0
+      - uses: microsoft/setup-msbuild@v1.1.3
```
Comment (michalvavrik) [inline .github/workflows/daily.yaml:203]: Please see https://github.com/microsoft/setup-msbuild/tags, both ilammy/msvc-dev-cmd@v1 and microsoft/setup-msbuild@v1.1.3 are tags for https://github.com/microsoft/setup-msbuild/commit/34cfbaee7f672c76950673338facd8a73f637506. Why is this change needed?

File: .github/workflows/pr.yaml [hunk 3]
```diff
@@ -107,7 +107,7 @@ jobs:
         shell: bash
         run: tar -xzf maven-repo.tgz -C ~
       - id: files
-        uses: jitterbit/get-changed-files@v1
+        uses: tj-actions/changed-files@v34.4.3
```
Comment (michalvavrik) [inline .github/workflows/pr.yaml:110]: Could you stick with Major?

Could you describe why is this change needed?

**Review by michalvavrik — CHANGES_REQUESTED** [review]

File: .github/workflows/daily.yaml (hunk 2 above)
Comment (afalhambra) [reply .github/workflows/daily.yaml:203]: same reasoning as before, but for this GHA.

File: .github/workflows/pr.yaml (hunk 3 above)
Comment (afalhambra) [reply .github/workflows/pr.yaml:110]: There hasn't been any [activity](https://github.com/jitterbit/get-changed-files) for this GHA `jitterbit/get-change-files` for more than 1 year and a half, and it's about to get deprecated due to Node12 not being updated. So I decided to get a more updated and active GHA replacement.

File: .github/workflows/daily.yaml (hunk 1 above)
Comment (michalvavrik) [reply .github/workflows/daily.yaml:202]: You say we should switch from https://github.com/ilammy/msvc-dev-cmd/commit/7315a94840631165970262a99c72cfb48a65d25d to https://github.com/ilammy/msvc-dev-cmd/commit/7315a94840631165970262a99c72cfb48a65d25d because v1 is same as v.12.0

File: .github/workflows/daily.yaml (hunk 2 above)
Comment (michalvavrik) [reply .github/workflows/daily.yaml:203]: I say you are actually not changing anything, just saying we should stick with some minor version. Both tags target same commit.

File: .github/workflows/daily.yaml (hunk 1 above)
Comment (afalhambra) [reply .github/workflows/daily.yaml:202]: sorry, not sure if I get you... 

I'm just trying to update that GHA from `v1` to `v1.12.0` as latter is the one updated to use Node16

File: .github/workflows/daily.yaml (hunk 2 above)
Comment (michalvavrik) [reply .github/workflows/daily.yaml:203]: image

File: .github/workflows/daily.yaml (hunk 1 above)
Comment (afalhambra) [reply .github/workflows/daily.yaml]: thank you @michalvavrik, didn't spot it

Comment (afalhambra) [reply .github/workflows/daily.yaml]: done

File: .github/workflows/daily.yaml (hunk 2 above)
Comment (afalhambra) [reply .github/workflows/daily.yaml]: thank you @michalvavrik, didn't spot it.
Done

**Review by michalvavrik — COMMENTED** [review]
Please check Windows failure: _##[error]Unexpected HTTP response: 404_

**Review by rsvoboda — APPROVED** [review]
We should be using GraalVM 22.3, but that can be handled in follow-up PR

**Review by michalvavrik — APPROVED** [review]
Thanks for changes.

**Comment by afalhambra** [comment]
> We should be using GraalVM 22.3, but that can be handled in follow-up PR

@rsvoboda I already opened it this morning https://github.com/quarkus-qe/beefy-scenarios/pull/308 but I was waiting for this PR to be merged first.

So will rebase that one later today or tomorrow
